drivers/net/wireless/cnss2/debug.h

```diff
@@ -23,27 +23,27 @@ extern void *cnss_ipc_log_long_context;
 	} while (0)
 
 #define cnss_pr_err(_fmt, ...) do {					\
-		printk("%scnss: " _fmt, KERN_ERR, ##__VA_ARGS__);	\
+		pr_err("%scnss: " _fmt, KERN_ERR, ##__VA_ARGS__);	\
```

Comment:
Uh oh, now the first character is `KERN_ERR` (etc. below) when that is already covered by `pr_err`.

Reply:
Right, double errors aren't fun. I reckon I should simply remove these?

Reply:
I don't understand what error you're facing. For cleanliness qcom should have used `printk(KERN_ERR "cnss: " _fmt, ##__VA_ARGS__);` but that's exactly what `pr_err` is defined to: https://elixir.bootlin.com/linux/v4.19/source/include/linux/printk.h#L308

Your only problem is with `pr_debug` which is toggled based on compile-time defines.

Reply:
It's also funny because the original qcom macro isn't taking `#define pr_fmt(fmt)` into account, while going through `pr_{debug,err,...}` does :grimacing:

Reply:
It's also funny because the original qcom macro isn't taking `#define pr_fmt(fmt)` into account, while going through `pr_{debug,err,...}` does :grimacing: